doc/README.md

```diff
@@ -0,0 +1,16 @@
+# API Documentation
+
+This is documentation for project API. It includes description of
+application endpoints that are current available. There are:
+
+- [login.md](login.md) and [logout.md](logout.md) endpoints
+- resource for accessing [users.md](users.md) - entities that
+represent users of application
+- resource for accessing [userprofile.md](userprofile.md) - data of authorized user
```

Comment:
It might be better to replace unfriendly ***userprofile.md*** with more informative ***current user information***